docker_enforcer.py

```diff
@@ -252,6 +252,7 @@ def make_container_periodic_check_compatible(cont_json, url, owner):
     cont_json["Name"] = "<unnamed_container>" if "name" not in url_params else url_params["name"][0]
     cont_json["Config"] = {}
     cont_json["Config"]["Labels"] = cont_json.get("Labels", [])
+    cont_json["HostConfig"] = {} if "HostConfig" not in cont_json else cont_json["HostConfig"]
```

Comment:
does it really make sense to add empty `HostConfig` here? maybe we should just refuse and return error? I think that docker CLI always puts some reasonable defaults there, even if default parameters are used. Other API clients should do as well, right? I don't remember now from where do we have this example of non-existing `HostConfig`...?